MVC/models.js

```diff
@@ -9,17 +9,21 @@ SELECT * FROM topics`).then(({ rows }) => {
 
 function fetchArticleId(article_id) {
 
-    return db.query(`
-SELECT * FROM articles WHERE article_id = $1;`, [article_id])
+    return db.query(`SELECT articles.*,
+    COUNT (comments.article_id)::int AS comment_count
+    FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id
+    WHERE articles.article_id = $1
+    GROUP BY articles.article_id`, [article_id])
         .then(({ rows }) => {
             if (rows.length === 0) { return Promise.reject({ status: 404, message: "The id provided does not exist!" }) }
+            console.log(rows)
             return rows[0]
         })
 }
 
 function fetchArticles(topic) {
     let sqlQuery = `
-SELECT articles.article_id, articles.title, articles.topic, articles.author, articles.created_at, articles.votes, articles.article_img_url,
+SELECT articles.*,
```

Comment:
I know this is earlier functionality but I just spotted it - did we want to change this to get all the columns back? I believe we didn't want the body to come back here.